scout/constants/file_types.py

```diff
@@ -2,11 +2,13 @@
 
 FILE_TYPE_MAP = {
     "vcf_cancer": {"category": "cancer", "variant_type": "clinical"},
```

Comment:
Something like this?

```suggestion
    "vcf_cancer": {"category": "cancer", "variant_type": "clinical", "loading_order":0},
    "vcf_cancer_research": {"category": "cancer", "variant_type": "research", "loading_order":1},
    ...
```

Reply:
Mm, yes, or perhaps a priority. It will seem a bit clunky, but at least very clear that there is a set order. I'll take a look.